pkg/operators/wasm/maps.go

```diff
@@ -209,6 +225,9 @@ func (i *wasmOperatorInstance) mapUpdate(ctx context.Context, m wapi.Module, sta
 	valuePtr := stack[2]
 	flags := stack[3]
 
+	getFD := (flags & getFdFlag) != 0
+	flags &= ^getFdFlag
```

Comment:
I really got confused by the `^`. TIL in golang bitwise not is `^`.
Apparently you can even do
```suggestion
	flags &^= getFdFlag
```